doc/source/methods/ALE.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<ul><li>Here I would say _replace the first feature value_ simply _replace $x_1$ with_ as that's easier to parse.</li><li>Just to make it clear that $x_{\1}^{(i)}$ means we leave all the other features unchanged? In which case it would be worth adding _and leave the rest of the features unchanged_ to make it easier to read quickly?</li><li>For c1, is calculated quanity = ALE(x1)? If you can it would be nice to discuss the different options for estimating this expectations (from a practical standpoint) as I have seen the code does consider some options.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<ul><li>For calculating c1 I didn't think it's worth going into detail as it's more of a small thing. Agree that calculated quantity is a big ambiguous, but I haven't given it a name (it's the first term in ALE(x1)^). I don;t think it's a big deal as the paragraph describing the calculation proceeds in a linear fashion so you end up with a "calculated quantity".</li></ul>